syft/frameworks/torch/hook/hook.py

```diff
@@ -1134,3 +1135,47 @@ def optim_float_precision_(optim_self):
             return optim_self
 
         self.torch.optim.Optimizer.float_precision = optim_float_precision_
+
+        # Modification of torch/nn/utils/clip_grad.py
+        def clip_grad_norm_remote_(parameters, max_norm, worker=None, norm_type=2):
+            r"""Clips gradient norm of an iterable of parameters stored over a remote model
+        
+            The norm is computed over all gradients together, as if they were
+            concatenated into a single vector. Gradients are modified in-place.
+        
+            Arguments:
+                - parameters (Iterable[Tensor] or Tensor): an iterable of PySyft remote
+                Tensors or a single Tensor that will have gradients normalized
+                - max_norm (float or int): max norm of the gradients
+                - worker: The worker where the parameters are hosted and where the gradient clipping
+                will be performed
+                - norm_type (float or int): type of the used p-norm. Can be ``'inf'`` for
+                    infinity norm.
+        
+            Returns:
+                Total norm of the parameters (viewed as a single vector).
+            """
+            if isinstance(parameters, torch.Tensor):
+                parameters = [parameters]
+            parameters = list(filter(lambda p: p.grad is not None, parameters))
+            max_norm = float(max_norm)
+            norm_type = float(norm_type)
+            if norm_type == inf:
+                total_norm = max(p.grad.data.abs().max() for p in parameters)
+            else:
+                # Init tensor with value 0 and dimension 1 (plain scalar)
+                total_norm = torch.Tensor(1)
```

Comment:
Could you replace torch.Tensor(1) by torch.zeros(1)?

Reply:
Done!